runtime/runtime/src/lib.rs

```diff
@@ -1436,6 +1439,21 @@ impl Runtime {
             let node_counter_after = state_update.trie().get_trie_nodes_count();
             tracing::trace!(target: "runtime", ?node_counter_before, ?node_counter_after);
 
+            let recorded_storage_diff = state_update
```

Comment:
I would also measure total adjusted storage size (+ total, + measure diff?) because this is our target metric, estimating how much data can be sent over network.

Reply:
Added chunk-level metrics. One measures the real recorded size (post finalization), the other records out upper bound estimation (pre-finalization). There's also a third metric to measure the difference between them.